contracts/LinniaPermissions.sol

```diff
@@ -120,6 +130,32 @@ contract LinniaPermissions is Ownable, Pausable, Destructible {
         return true;
     }
 
+    /// Give a viewer access to a linnia record
+    /// Called by owner of the record.
+    /// @param dataHash the data hash of the linnia record
+    /// @param viewer the user being granted permission to view the data
+    /// @param dataUri the path of the re-encrypted data
+    function grantPolicyBasedAccess(
```

Comment:
Should client's choose whether or not they conform to a policy when they grant permission? Or should a policy be enforced universally for a record?

Reply:
I would think the record owner would want to set a policy for their record when they upload it, not on a permission by permission basis

Reply:
This approach leaves permissioning up to Andrea though, not the person who owns the record itself

Reply:
And if Andrea is the only stakeholder here, this could be done off chain